### ISSUE TITLE
External link background in dark mode

### ISSUE THREAD

**Issue by iaian** [freeCodeCamp/devdocs#892]
Noticed an issue with external links dark mode:
<img width="827" alt="screen shot 2018-10-09 at 11 47 09 am" src="https://user-images.githubusercontent.com/569378/46687850-1b2b2200-cbb9-11e8-916d-471a61080b30.png">

To reproduce:
* Clear cache
* Enable dark mode
* Visit page with external link, ex : https://devdocs.io/vue~2/guide/instance
* See that the external link has a background image across its area

Issue originates from css rule:
```
html._theme-dark ._list-link, html._theme-dark ._page a[href^="http:"], ._page html._theme-dark a[href^="http:"], html._theme-dark ._page a[href^="https:"], ._page html._theme-dark a[href^="https:"] {...}
```

Happy to open a PR to fix if that is welcome!
Thanks for your work on this tool 👍

**Comment by j-f1** [comment]
Feel free to open a PR!

**Comment by iaian** [comment]
PR : https://github.com/freeCodeCamp/devdocs/pull/893

**Comment by Thibaut** [comment]
Fixed by d72de330aa4086297095f4c7018c19263d07335e. Thanks for the bug report!
